### DISCUSSION TITLE
this is a new discussion originated by a user

### DISCUSSION THREAD

**Discussion by anton-vrba** [WinTestEdt/WinTestEdt#6] [1.3 Ideas and Feature requests]
a new discussion

**Comment by WinTestEdt** [comment]
Thanks for this


------ Original Message ------
From "anton-vrba" ***@***.***>
To "WinTestEdt/WinTestEdt" ***@***.***>
Cc "Subscribed" ***@***.***>
Date 1/26/2024 9:20:54 PM
Subject [WinTestEdt/WinTestEdt] this is a new discussion originated by a 
user (Discussion #6)
